app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -20,8 +20,8 @@ def schedule_in_person_enrollment(user, pii)
         analytics(user: user).usps_ippaas_enrollment_created(
           enrollment_code: enrollment.enrollment_code,
           enrollment_id: enrollment.id,
-          user_id: enrollment.user_id,
-          service_provider: enrollment.service_provider,
+          user_id: user.uuid,
+          service_provider: enrollment.issuer,
```

Comment:
```suggestion
          service_provider: enrollment.service_provider.issuer,
```
We don't need to explicitly pass `user_id` because we're passing the user to `Analytics` (see [this line](https://github.com/18F/identity-idp/blob/main/app/services/analytics.rb#L28) in the Analytics service).

And, the issuer is a property of the service provider object on the enrollment.

Reply:
We can also remove the `user_id` parameter from the `usps_ippaas_enrollment_created` method in `AnalyticsEvents`.